starknet_py/net/gateway_client.py

```diff
@@ -210,15 +214,13 @@ async def estimate_fee(
 
     async def call_contract(
         self,
-        invoke_tx: Union[InvokeFunction, Call],
+        call: Call = None,  # pyright: ignore
```

Comment:
Same here. I think it is intentional, but would like to know

Reply:
Yes, it's intentional.

Reply:
I wanted the interface to enforce using `calls`. Existing implementations still support using `invoke_tx` for backwards compatibility until we deprecate it completely.